### PR TITLE
Robust test helper

### PR DESCRIPTION
**Pull request by Robdel12**
## What is this? 

This takes the test helper that was merged in #14 and makes it a little more robust. You now don't need to import it into your test helper, we do that for you. We also include a generator to add `select` to the `.jshintrc` file. Finally you now can use the select helper to select multipleselects too. 

## TODO:
- [x] Remove initializer and use blueprints to add helper imports
- [x] Add docs regarding generator adding the helper imports
- [x] Move test helper to `test-support` folder.
- [x] Test this all in a dummy project.

### CODE REVIEW

File: app/initializers/register-select-helper.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import ENV from '../config/environment';
+import registerSelectHelper from 'emberx-select/helpers/register-select-helper';
```
Comment (cowboyd) [inline app/initializers/register-select-helper.js]: This should probably be in `test-support` or some other package that isn't included into the app in production. I don't really know how to do that though. Maybe @fivetanley does?

File: blueprints/emberx-select/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  description: '',
+
+  normalizeEntityName: function() {},
+
+  // locals: function(options) {
+  //   // Return custom template variables here.
+  //   return {
+  //     foo: options.entity.options.foo
+  //   };
+  // }
```
Comment (cowboyd) [inline blueprints/emberx-select/index.js]: Let's just delete the commented code.

File: app/initializers/register-select-helper.js (hunk 1 above)
Comment (trabus) [inline app/initializers/register-select-helper.js]: I think you can do this with [broccoli-stew's rm](https://github.com/stefanpenner/broccoli-stew/blob/master/lib/rm.js#L40) in an addon hook: http://stackoverflow.com/questions/30686368/exclude-folders-from-builds-in-brocfile/30741440#30741440

**Comment by Robdel12** [comment]
Merging this. @tehviking gave his verbal :+1:

**Comment by cowboyd** [comment]
Sorry never got back to it. What was the reason for not using an initializer to auto-register? Unless it's totally not approved by the community, it seems better than generating boilerplate.

**Comment by Robdel12** [comment]
@cowboyd I looked at [201-created's test helper](https://github.com/201-created/ember-cli-acceptance-test-helpers) lib and [Torn's test helper](https://github.com/toranb/ember-cli-test-helpers) lib for guidance. There wasn't a straight forward way of removing the initializer from the postbuild process so it doesn't end up in production.

**Comment by cowboyd** [comment]
Awesome. Glad it's merged then!

**Pull request merged**